addons/web/static/src/legacy/js/fields/field_utils.js

```diff
@@ -367,7 +373,7 @@ function formatMonetary(value, field, options) {
     if (options.forceString) {
         return val.join(' ');
     }
-    return utils.Markup(val.map((v) => _.escape(v)).join('&nbsp;'));
+    return utils.Markup(val.map((v) => _.escape(v)).join(NBSP));
```

Comment:
_Line may contain security risk: Markup_

Reply:
You can just remove the Markup call here, since there will no longer be any HTML in the string

Reply:
I don't understand the purpose of corresponding commit then https://github.com/odoo/odoo/commit/e89b626f5411b7836030c6daa940b41f65747843

cc @xmo-odoo

Reply:
I think there used to be a t-out or t-raw that used the value returned by this so that the non-breaking spaces were preserved, but since the non-breaking spaces are now just unicode characters it's no longer necessary. The commit in question is part of a series of commit whose purpose was to remove t-raw and use t-out instead (which html-espaces strings but not markup objects).

Reply:
I'd avoid code clean up in v15. Can't we remove Markup call in master instead?

Reply:
Sure, but in that case you need a security override ;) @odoo/rd-security